ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -4,6 +4,9 @@
 //
 
 // Accessibility
+"Accessibility.ActivityIndicator.Animating.label" = "In progress";
+"Accessibility.ActivityIndicator.Stopped.label" = "Progress halted";
```

Comment:
"Progress Halted" seems weird. Did we get that string from somewhere? What does normal UIActivityIndicator say?

Reply:
yes it is from apple.